app/models/test_result.rb

```diff
@@ -5,4 +5,17 @@ def readonly?
     true
   end
 
+  def self.slowest
+    select('name, AVG(runtime) AS average_runtime').
+      where(status: 'passed').
+      group(:name).
+      order('average_runtime DESC')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.